settings.gradle.kts

```diff
@@ -267,48 +256,59 @@ include(":instrumentation:hikaricp-3.0:library")
 include(":instrumentation:hikaricp-3.0:testing")
 include(":instrumentation:http-url-connection:javaagent")
 include(":instrumentation:hystrix-1.4:javaagent")
+include(":instrumentation:internal:internal-application-logger:bootstrap")
+include(":instrumentation:internal:internal-application-logger:javaagent")
+include(":instrumentation:internal:internal-class-loader:javaagent")
+include(":instrumentation:internal:internal-class-loader:javaagent-integration-tests")
+include(":instrumentation:internal:internal-eclipse-osgi-3.6:javaagent")
+include(":instrumentation:internal:internal-lambda-java9:javaagent")
+include(":instrumentation:internal:internal-lambda:javaagent")
+include(":instrumentation:internal:internal-reflection:javaagent")
+include(":instrumentation:internal:internal-reflection:javaagent-integration-tests")
+include(":instrumentation:internal:internal-url-class-loader:javaagent")
+include(":instrumentation:internal:internal-url-class-loader:javaagent-integration-tests")
 include(":instrumentation:java-http-client:javaagent")
 include(":instrumentation:java-http-client:library")
 include(":instrumentation:java-http-client:testing")
 include(":instrumentation:java-util-logging:javaagent")
 include(":instrumentation:java-util-logging:shaded-stub-for-instrumenting")
-include(":instrumentation:jaxrs:jaxrs-common:bootstrap")
-include(":instrumentation:jaxrs:jaxrs-common:javaagent")
-include(":instrumentation:jaxrs:jaxrs-common:testing")
+include(":instrumentation:jaxrs-client:jaxrs-client-1.1-testing")
+include(":instrumentation:jaxrs-client:jaxrs-client-2.0-testing")
```

Comment:
In my opinion it would be more natural to have `jaxrs-client` after `jaxrs`, that is also how intellij displays them

Reply:
> In my opinion it would be more natural to have `jaxrs-client` after `jaxrs`, that is also how intellij displays them

I adjusted all modules order according to Intellij now, we may pay more attention to relevant content in future PRs. otherwise, it's time consuming to amend:（